.env.example

```diff
@@ -0,0 +1,6 @@
+export AWS_ACCESS_KEY_ID=YOUR_ACESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=YOUR_ACCESS_KEY
```

Comment:
should we remove env variables we're not using yet?

Reply:
fixed